src/http-api/index.js

```diff
@@ -0,0 +1,37 @@
+var Hapi = require('hapi')
+var IPFS = require('../ipfs-core')
+var debug = require('debug')
+var log = debug('api')
+
+exports = module.exports
+
+exports.start = function (callback) {
+  // start IPFS and exports.ipfs = new IPFS()
+
+  exports.ipfs = new IPFS()
+
+  var server = exports.server = new Hapi.Server({
+    connections: {
+      routes: {
+        cors: true
+      }
+    }
+  })
+
+  server.connection({
+    port: 9000
+  })
+
+  // load routes
+  require('./routes/version.js')
```

Comment:
This could be much cleaner:

```js
server.route(require('./routes/version.js')

// version.js
var resources = require('./../resources')

module.exports = {
  method: 'GET',
  path: '/api/v0/version',
  handler: resources.version.get
}
```